List orders by date and toPayment() (#54)
intergiro/developer-portal: docs/merchant/customer/payment-methods.md

}
```

## Selecting Customer Method

> Use the function `Customer.Method.toPayment()` in the repository <a target="_blank" href="https://github.com/intergiro/model">Model</a> to convert a customer method to a [card payment](../order/reference.html#card-payment). Put the [card payment](../order/reference.html#card-payment) on the [Order Creatable](../order/reference.html#creatable) and POST to the [order endpoint](../order/create.html).

Reviewer: ```suggestion
Use the function `Customer.Method.toPayment()` from the repository <a target="_blank" href="https://github.com/intergiro/model">Model</a> to convert a customer method to a [card payment](../order/reference.html#card-payment). Put the [card payment](../order/reference.html#card-payment) on the [Order Creatable](../order/reference.html#creatable) and POST to the [order endpoint](../order/create.html).
```

Reviewer: Also the maybe the link should go to npm 
https://www.npmjs.com/package/@payfunc/model

> Use the function `Customer.Method.toPayment()` in the repository <a target="_blank" href="https://github.com/intergiro/model">Model</a> to convert a customer method to a [card payment](../order/reference.html#card-payment). Put the [card payment](../order/reference.html#card-payment) on the [Order Creatable](../order/reference.html#creatable) and POST to the [order endpoint](../order/create.html).

Reviewer: ```suggestion
To select a customer method, use the function `Customer.Method.toPayment()` from the repository <a target="_blank" href="https://github.com/intergiro/model">Model</a> to convert a customer method to a [card payment](../order/reference.html#card-payment). Then, put the [card payment](../order/reference.html#card-payment) on the [Order Creatable](../order/reference.html#creatable) and POST to the [order endpoint](../order/create.html).
```

Reviewer: Also the maybe the **Model** link should go to npm
https://www.npmjs.com/package/@payfunc/model

> To select a customer method, use the function `Customer.Method.toPayment()` from the repository <a target="_blank" href="https://github.com/intergiro/model">Model</a> to convert a customer method to a [card payment](../order/reference.html#card-payment). Then, put the [card payment](../order/reference.html#card-payment) on the [Order Creatable](../order/reference.html#creatable) and POST to the [order endpoint](../order/create.html).

Reviewer: Maybe use the npm link to **model** instead?
https://www.npmjs.com/package/@payfunc/model

> 		})

Reviewer: ```suggestion
})
```
